examples/medplum-provider/src/pages/encounter/EncounterChart.tsx

```diff
@@ -201,6 +201,39 @@ export const EncounterChart = (): JSX.Element => {
     [tasks]
   );
 
+  const saveChargeItem = useCallback(
+    async (chargeItem: ChargeItem): Promise<ChargeItem> => {
+      try {
+        return await medplum.updateResource(chargeItem);
+      } catch (err) {
+        showNotification({
+          color: 'red',
+          icon: <IconCircleOff />,
+          title: 'Error',
+          message: normalizeErrorString(err),
+        });
+        return chargeItem;
+      }
+    },
+    [medplum]
+  );
+
+  const updateChargeItemList = useCallback(
+    (updatedChargeItem: ChargeItem): void => {
+      if (saveTimeoutRef.current) {
+        clearTimeout(saveTimeoutRef.current);
+      }
+
+      saveTimeoutRef.current = setTimeout(async () => {
+        const savedChargeItem = await saveChargeItem(updatedChargeItem);
+        setChargeItems(chargeItems.map((item) => (item.id === savedChargeItem.id ? savedChargeItem : item)));
```

Comment:
Do we need to call this again?  It seems like this will do the same thing as the call on line 232?